### ISSUE TITLE
Webm display above album download selector

### ISSUE THREAD

**Issue by beschoenen** [Floens/Clover#162]
When playing webms and clicking download album, the video will display about the download selector view

**Comment by Floens** [comment]
Yeah I've seen this problem before. It's because the VideoView is actually a no-op view, and Android places the video on top of everything to make hardware acceleration easier. VideoView doesn't support  support the visibility settings and as such seems to appear above everything.

**Comment by floens** [comment]
Fixed with ExoPlayer.
